### DISCUSSION TITLE
Installation - Composer conflict

### DISCUSSION THREAD

**Discussion by mathamel** [lunarphp/lunar#1926] [Q&A]
Hi, I try to install lunar to my laravel 11 app. (11.21.0) - PHP 8.2.22

I try with lunarphp/lunar (0.8.1)

It says compatible with :
PHP >= 8.2
Laravel 10 / Laravel 11
MySQL 8.0+ / PostgreSQL 9.2+

Yet, with composer install 

`Problem 1
    - lunarphp/core 0.8.1 requires laravel/framework ^9.0|^10.0 -> found laravel/framework[v9.0.0, ..., v9.52.16, v10.0.0, ..., v10.48.20] but it conflicts with your root composer.json require (^11.9).
    - lunarphp/lunar 0.8.1 requires lunarphp/core 0.8.1 -> satisfiable by lunarphp/core[0.8.1].
    - Root composer.json requires lunarphp/lunar v0.8.1 -> satisfiable by lunarphp/lunar[0.8.1].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.`

Seems lunar not really compatible with Laravel 11

**Comment by glennjacobs** [comment]
You will want to install the v1 beta for Laravel 11. That will require you to update your app's composer.json to set "minimum-stability": "dev".
